worker/notify.go

```diff
@@ -10,6 +10,15 @@ import (
 	"time"
 )
 
+func closeConnection(conn *grpc.ClientConn) {
+	func(conn *grpc.ClientConn) {
+		err := conn.Close()
+		if err != nil {
+			log.WithError(err).Error("Could not close connection")
+		}
+	}(conn)
+}
```

Comment:
This should do the same

```suggestion
func closeConnection(conn *grpc.ClientConn) {
	err := conn.Close()
	if err != nil {
		log.WithError(err).Error("Could not close connection")
	}
}
```

Reply:
Thanks, I forgot to remove it